go.mod

```diff
@@ -11,9 +11,9 @@ require (
 )
 
 require (
+	github.com/google/go-cmp v0.5.7 // indirect
```

Comment:
I am typically against new dependencies unless they somehow are super hyper useful and necessary, ie needs a justification (good part though is at least it's not a package from a rando :)  and it's indirect so... )

is this coming from  httptest.NewRecorder() maybe ? (I can do `go mod why` I guess)